iphone/Classes/TiUITableView.m

```diff
@@ -2053,6 +2053,16 @@ - (void)setContentInsets_:(id)value withObject:(id)props
   }
 }
 
+- (void)setAllowsMultipleSelectionDuringEditing_:(id)value
+{
+  ENSURE_TYPE(value, NSNumber);
+  [[self proxy] replaceValue:value forKey:@"allowsMultipleSelectionDuringEditing" notification:NO];
+
+  [[self tableView] beginUpdates];
```

Comment:
Why do we need `beginUpdates` and `endUpdates` here? We are not modifying rows/sections of the table view, are we?